templates/networking/resources/vkcs_networking_floatingip.md.tmpl

```diff
@@ -18,7 +18,11 @@ description: |-
 If one of the subnets in a list has an exhausted pool, terraform will try the
 next subnet ID from the list.
 
-{{tffile "templates/networking/resources/vkcs_networking_floatingip/subnet_list/main.tf"}}
+{{tffile "examples/networking/floatingip/main-subnets.tf"}}
```

Comment:
i am not sure we want this case

Reply:
im pretty sure that its a good example but may be with not pre-created pool and network. the user should be able to specify in which range he wants a floatingip so specifying subnet or list of subnets can help him achieve this. 
but then i will nee to create a pool here with several subnets